rotkehlchen/exchanges/coinbase.py

```diff
@@ -598,14 +603,17 @@ def _deserialize_asset_movement(self, raw_data: Dict[str, Any]) -> Optional[Asse
             # Only get address/transaction id for "send" type of transactions
             address = None
             transaction_id = None
+            transaction_url = raw_data['id']
```

Comment:
This and all other places where dict is accessed without `get()` may raise KeyError.

Also is the `'id'` the url? Or did you mean another field?

Perhaps since we used it in the link field? The link field for trades that are from exchanges generally contain a unique identifier. So it's nor really a url. 

Or perhaps since you use the same variable later to get the transaction_url from 'network' object if it exists?

Reply:
'id' was left for the initial implementation.  If it has from, and the from/to resource is network, it will check the network and get a blockchain.info url or etherscan.io url contained in the response.  This may affect both network withdrawals and deposits.